languages/ruby/exercises/concept/ostruct/.docs/introduction.md

```diff
@@ -0,0 +1,33 @@
+Ruby comes with a Standard Library - a collection of classes for working with things such as dates, json, and networking.
+The also provide some useful functionality for making your code easier to work with.
+
+`OpenStruct` is part of the standard library and allows you to easily create an object from a hash. Rather than having to access using hash keys, `OpenStruct` instead allows us to use methods to access and set values.
+
+When using classes from the Standard Library, or any other library, you need to require that class using the `require` method.
+
+```ruby
+require 'ostruct'
+
+attributes = { name: "Jeremy Walker", age: 21, location: "Nomadic" }
+person = OpenStruct.new(attributes)
+
+person.name
+#=> Jeremy Walker
+
+person.location
+#=> Nomadic
+
+# Update the age
+person.age = 35
+
+# It sets correctly
+person.age
+#=> 21
```

Comment:
This seems to be incorrect, as the setter should have set it to 35, I think.  Is the comment "stale"?

Reply:
```suggestion
#=> 35
```